doc/manuals/admin/yum.md

```diff
@@ -0,0 +1,49 @@
+## Using YUM repositories
+
+There are 2 available packages:
+- nightly 
+- release
+ 
+You can manually add config for yum repo or download it from FIWARE public repository.
+
+Use this config for nightly packages:
+```
+[fiware-nightly]
+name=FIWARE nightly repository
+baseurl=http://nexus.lab.fiware.org/repository/el/$releasever/$basearch/nightly
```

Comment:
I have tryied to access to http://nexus.lab.fiware.org/repository/el/7/x86_64/nightly and I get a 404 error.

Not sure if this is the expected behaviour (in the all repository, when I access to http://repositories.lab.fiware.org/repo/rpm/6/x86_64/ I get the list of packages...)

Reply:
You can't browse nexus.lab.fiware.org/repository/el directly, use this link: https://nexus.lab.fiware.org/service/rest/repository/browse/el/,

otherwise you have to enter direct paths, like:
https://nexus.lab.fiware.org/repository/el/7/x86_64/nightly/contextBroker-1.12.0-20180227.x86_64.rpm

Reply:
So, current baseurl= is ok, rigth?

Just to be sure, to have an explicit confirmation of the expert (i.e. you ;)

Reply:
Yes, ok. Byt the way, thanks!